util/bisect

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+module RefinedPathExpander
+  def expand_dirs_to_files(*dirs)
+    super.sort
+  end
+end
+
+require 'path_expander'
+
+class PathExpander
```

Comment:
Is this the only change needed to successfully run `minitest_bisect`?

Reply:
`minitest_bisect` runs fine without this. The problem is that `rake` and `path_expander` (`minitest-bisect`'s file loader) currently load files differently. So, they shuffle a different array when randomizing. Thus, failures reproducible through `rake` don't reproduce through `minitest_bisect` even if the same seed is used.

Reply:
Are you on Linux or something such that `Dir["*"]` isn't sorted? Is rake explicitly sorting?

Reply:
Yes, I'm on Linux. Rake explicitly sorts [here](https://github.com/ruby/rake/blob/67ced2fab148db0ada6b0d52d321f5cc86e22960/lib/rake/file_list.rb#L404-L409), so for example in rubygems I get

```
> irb(main):001:0> FileList['test/**/test_*.rb']
=> ["test/rubygems/test_bundled_ca.rb", "test/rubygems/test_config.rb", "test/rubygems/test_deprecate.rb", "test/rubygems/test_gem.rb", "test/rubygems/test_gem_available_set.rb", ..., "test/rubygems/test_gem_version.rb", "test/rubygems/test_gem_version_option.rb", "test/rubygems/test_kernel.rb", "test/rubygems/test_remote_fetch_error.rb", "test/rubygems/test_require.rb"]

irb(main):002:0> Dir['test/**/test_*.rb']
=> ["test/rubygems/test_gem_commands_info_command.rb", "test/rubygems/test_gem_resolver_vendor_set.rb", "test/rubygems/test_gem_dependency_list.rb", "test/rubygems/test_gem_source_fetch_problem.rb", "test/rubygems/test_gem_request.rb", "test/rubygems/test_gem_resolver_best_set.rb", ..., "test/rubygems/test_gem_gem_runner.rb", "test/rubygems/test_gem_resolver_local_specification.rb", "test/rubygems/test_gem_available_set.rb", "test/rubygems/test_gem_security_policy.rb", "test/rubygems/test_gem_security.rb", "test/rubygems/test_gem_impossible_dependencies_error.rb"]
```